spec-main/api-safe-storage-spec.ts

```diff
@@ -12,8 +12,43 @@ import * as fs from 'fs';
 *
 * Because all encryption methods are gated by isEncryptionAvailable, the methods will never return the correct values
 * when run on CI and linux.
+* Refs: https://github.com/electron/electron/issues/30424.
 */
 
+describe('SafeStorage.isEncryptionAvailable() before and after app is ready', () => {
+  it('isEncryptionAvailable() returns false before app is ready', async () => {
+    if (!app.isReady()) {
```

Comment:
This test is meaningless because this function will never return true.

Reply:
@nornagon do you suggest removing this line and the matching `}`? I though there might be some race condition where `app.isReady()` might return `true` even before we do `await app.whenReady()` which I'm already doing in the rest of the test after this if block ends. Not sure why you think the entire test is meaningless.

Reply:
If you actually want to test what happens when the app isn't ready, you need to launch a fresh instance of Electron.

Reply:
In that case we should turn this test into an app inside fixtures and run that as a child process from here?

Reply:
Indeed!

Reply:
Done!